tests/test_login_functionality.py

```diff
@@ -20,6 +20,7 @@ def test_login_functionality(self):
 
         username_field = driver.find_element(By.XPATH, '//input[@data-test="username"]')
         username_field.send_keys(USERNAME)
+        # TODO: add step for checking username field value
         assert username_field.is_displayed(), "The username is not visible."
```

Comment:
@Sv-24 in the end you could check some Page header, it is better way.
Pls, delete unnecessary code